packages/forma-36-website/src/content/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Getting started'
+---
+
+## Installation
+
+### Install components
+
+Inside your React project directory, install Forma 36 by running either of the following:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-components
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-components`
+```
+
+#### Installing components as separate packages
+
+ou can install Forma 36 React components as separate packages if that's what you need for your project. For example, here's how to install just the button:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-button
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-button`
+```
+
+The above option is available, but our recommendation is to use the `@contentful/f36-components` package. Tree shaking will make your bundle contain only components that you actually import and need.
```

Comment:
[Suggestion]
Merge this with the opening paragraph of the section

Reply:
Why do we recommend this? Wouldn't installing components as separate packages — therefore a smaller bundle size — be a good thing?

Reply:
We recommend using `f36-components` because tree shaking should take care of the bundle size. Even if you install the full package, and then use only one component, tree shaking will make only this one component end up in your bundle. 
But we still want to provide an option to import separate packages, users might have some reasons to do that.